.gitignore

```diff
@@ -23,4 +23,4 @@ packages/*/lib
 storybook
 
 # Coverage
-coverage
```

Comment:
@devethan @DevYuns @smallbee3 Here is the reason why we should render `white space` in `vscode`.

Reply:
@hyochan what does `white space` mean?